static/js/settings_org.js

```diff
@@ -813,12 +813,38 @@ exports.build_page = function () {
         var subsection_id = save_button.attr('id').replace("org-submit-", "");
         var subsection = subsection_id.split('-').join('_');
 
+        var properties_changed = [];
+        var properties_elements = get_subsection_property_elements($(e.target).closest('.org-subsection-parent'));
+        _.each(properties_elements, function (elem) {
+            if (check_property_changed(elem)) {
+                var property_name = exports.extract_property_name($(elem));
+                property_name = property_name.replace("realm_", "");
+                property_name = property_name.replace("_minutes", "_seconds");
+                property_name = property_name.replace("msg_edit_limit_setting", "allow_message_editing");
+                property_name = property_name.replace("msg_delete_limit_setting", "allow_message_deliting");
+                properties_changed.push(property_name);
+            }
+        });
```

Comment:
This logic feels really brittle, and in any case basically duplicates some of the logic in `get_complete_data_for_subsection`.  I think a better implementation would instead be editing `get_complete_data_for_subsection` (e.g. having it call `get_complete_data_for_subsection` internally) and having it implement these details.

Reply:
Okay! I can implement that logic in `get_complete_data_for_subsection`. Is the way I'm extracting changed properties using `check_propery_changed` good?

Reply:
I'm not sure; I think you should coordinate a bit with @shubhamdhama and @pragatiagrawal31 as they are working on the organization settings save codebase and have more context (I had actually originally imagined @pragatiagrawal31 working on this issue)